.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # default PR reviews to the team
-* @hashicorp/consul
+* @hashicorp/consul-selfmanage-maintainers
 
 # release configuration
 /.release/                              @hashicorp/release-engineering
```

Comment:
This needs to be updated to `@hashicorp/team-selfmanaged-releng`

Reply:
Will do, and I'll add the request on passport too!